deployment/docker-compose.yml

```diff
@@ -67,8 +67,30 @@ services:
         max-size: "1m"
         max-file: "10"
 
+  # to run stack without backup, please comment this out
+  # currently supports only AWS s3. GCP and Azure TODO
+  postgres_backup:
+    build:
+      context: ./backuper
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    environment:
+    - NETWORK=${NETWORK:-testnet}
+    - CARDANO_NODE_SOCKET_PATH=/app/node-ipc/node.socket
+    - POSTGRES_DB=cardano
+    - PGPASSFILE=/run/secrets/.pgpass
+    - S3_BUCKET=${S3_BUCKET:?Please set me}
```

Comment:
Is it okay to leave this as "Please set me"?

Reply:
notation
${VARIABLE:?text}
will print "text" when variable is unset
Doesn't really matter what we put here, it is error message